services/repository-managers/src/test/java/org/sagebionetworks/repo/manager/EntityPermissionsManagerImplUnitTest.java

```diff
@@ -872,18 +914,42 @@ public void testHasDownloadAccessWihoutOpenDataAsAnonymous() {
 		when(mockNodeDao.getBenefactor(nodeId)).thenReturn(benefactorId);
 		when(mockObjectTypeManager.getObjectsDataType(nodeId, ObjectType.ENTITY)).thenReturn(dataType);
 
-		when(mockAclDAO.canAccess(userInfo.getGroups(), benefactorId, ObjectType.ENTITY, ACCESS_TYPE.DOWNLOAD)).thenReturn(false);
-		
 		// Call under test
 		AuthorizationStatus status = entityPermissionsManager.hasAccess(nodeId, ACCESS_TYPE.DOWNLOAD, userInfo);
 		
 		assertFalse(status.isAuthorized());
-		assertEquals("You lack DOWNLOAD access to the requested entity.", status.getMessage());
+		assertEquals("Your authorization scope(s) do not allow DOWNLOAD access.", status.getMessage());
```

Comment:
Might be confusing wording referring to scopes

Reply:
Note: this is an issue only for the anonymous user

Reply:
Probably also in this case the anonymous user can have the full scope.

Reply:
Giving full scope to the anonymous user fixed this.